v2/go.mod

```diff
@@ -2,9 +2,11 @@ module github.com/nats-io/jwt/v2
 
 go 1.18
 
-require github.com/nats-io/nkeys v0.3.1-0.20221205184623-5d8a6730c42c
+require (
+	github.com/nats-io/nkeys v0.3.1-0.20221205184623-5d8a6730c42c
```

Comment:
something unreleased from nkeys is needed?

Reply:
possibly that is xkeys - let me check what that is

Reply:
should we make an nkeys release? 
https://github.com/nats-io/nkeys/compare/v0.3.0...main

Reply:
yes this for the unreleased xkeys - we will need to release that when we release the lib